src/App/Fossa/VSI/IAT/Fingerprint.hs

```diff
@@ -40,7 +41,7 @@ hashFile fp =
   sendIO (runConduitRes (sourceFile fp .| sinkHash))
     `catch` (\(e :: IOException) -> fatalText ("unable to hash file: " <> toText (show e)))
 
-fingerprintRaw :: (Has (Lift IO) sig m, Has Diagnostics sig m) => Path Abs File -> m Fingerprint
+fingerprintRaw :: (Has (Lift IO) sig m, Has Diagnostics sig m) => Path x File -> m Fingerprint
```

Comment:
[nit] I'd like to keep this as `Path Abs File` if possible -- ensures that we're not accidentally using relative filepaths if, e.g., we run with a different `BaseDir` like `fossa analyze path/to/dir`